deployment/terraform/create.go

```diff
@@ -1063,11 +1064,58 @@ func (t *Terraform) updateAppConfig(siteURL string, sshc *ssh.Client, jobServerE
 			keycloakScheme = "http"
 		}
 
-		cfg.OpenIdSettings.Enable = model.NewPointer(true)
-		cfg.OpenIdSettings.ButtonText = model.NewPointer("Keycloak Login")
-		cfg.OpenIdSettings.DiscoveryEndpoint = model.NewPointer(keycloakScheme + "://" + t.output.KeycloakServer.PublicDNS + ":8080/realms/" + t.config.ExternalAuthProviderSettings.KeycloakRealmName + "/.well-known/openid-configuration")
-		cfg.OpenIdSettings.Id = model.NewPointer(t.config.ExternalAuthProviderSettings.KeycloakClientID)
-		cfg.OpenIdSettings.Secret = model.NewPointer(t.config.ExternalAuthProviderSettings.KeycloakClientSecret)
+		// Setup SAML certificate for Keycloak
+		samlIDPCert, err := os.Open(t.getAsset("saml-idp.crt"))
```

Comment:
This file is starting to get too long. Can we move these to a separate (terraform/keycloak.go)?